backend/templates/model-na-31-2.typ

```diff
@@ -0,0 +1,354 @@
+#import "common/style.typ": conf, title, mono
+#import "common/scripts.typ": alertbox, block_with_checkbox, date_input, time_input, letterbox_main, summary_table, text_input, format_date, TODO
+#let input = json("inputs/model-na-31-2.json")
+
+#show: doc => conf(input, doc)
+
+#let is_municipality = (municipal, public_body) => if input.election.category == "Municipal" [#municipal] else [#public_body]
+
+#title(
+  [Model Na 31-2],
+  [Proces-verbaal van een #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam] (centrale stemopneming)],
+  [
+    De verkiezing van de leden van *#input.election.name* \ op *#format_date(input.election.election_date)* \
```

Comment:
`election.name` doesn't really match what should be input here, is there something else we could display, or leave it as is?

Reply:
I think this should be based on the `election.category`